logging/logging.go

```diff
@@ -71,6 +69,15 @@ func IsDebugLevel(logger *log.Entry) bool {
 	return logger.Level == log.DebugLevel
 }
 
+// FormatterWrapper wraps log.Formatter interface and adds functions for customizations.
+// Intention for this interface is to provide ability to customize logging by accustomed:
+// `logging.SetServiceName` / `logging.SetHooks` from main function of Acra services
+type FormatterWrapper interface {
```

Comment:
I'd call it just `Formatter` since the users don't need to actively *wrap* anything into it. It's just that `logging.Formatter` is a `logrus.Formatter` with a couple of additional methods that we need. Acra formatters already implement this interface.

Nice use of interface for customization, by the way! 👍